testsuite/run-tests

```diff
@@ -12,11 +12,11 @@ if __name__ == '__main__':
         status = suite.results[k].name
         if status not in ('PASS', 'XFAIL'):
             all_ok = False
-            print "--- {} : {} ---".format(k, status)
+            print("--- {} : {} ---".format(k, status))
             with open(os.path.join(suite.output_dir,
                                    "{}.yaml".format(k)), 'rb') as f:
                 y = f.read()
-                print yaml.safe_load(y).out
+                print(yaml.safe_load(y).out)
 
     if all_ok:
-        print "SUCCESS"
+        print("SUCCESS")
```

Comment:
The testsuite framework now prints a summary automatically, and the `-E/--show-error-output` option also displays output/logs for failing tests, so you should be able to remove this and just leave:

```python
suite = testsuite.ALSTestsuite(os.path.dirname(__file__))
sys.exit(suite.testsuite_main())
```

Reply:
This bit doesn't work, `testsuite/run.sh` already has `--show-error-output` but doesn't print the output in case of failing tests.